includes/Utilities/Plugin_Request_Utility.php

```diff
@@ -40,12 +40,16 @@ class Plugin_Request_Utility {
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param string $plugin_slug The plugin slug or basename.
+	 * @param string $plugin_slug The plugin slug, basename or arbitrary path.
 	 * @return string The plugin basename.
 	 *
 	 * @throws Exception Thrown if an invalid basename or plugin slug is provided.
 	 */
 	public static function get_plugin_basename_from_input( $plugin_slug ) {
+		if ( is_dir( $plugin_slug ) ) {
+			return $plugin_slug;
+		}
```

Comment:
We should not need to change this method.

This change should happen in `Abstract_Check_Runner::get_plugin_basename()`

Reply:
I don't understand. This conditional jump the test if the plugin is activated and get actived plugins, that is necessary.

Reply:
This conditional should be done in `Abstract_Check_Runner::get_plugin_basename()`, not here.

Reply:
May be like this in `Abstract_Check_Runner::get_plugin_basename()`:
```
			if ( is_dir( $plugin ) ) {
				$this->plugin_basename = $plugin;
			} else {
				$this->plugin_basename = Plugin_Request_Utility::get_plugin_basename_from_input( $plugin );
			}

```